src/visualstates/gui/dialogs/librariesdialog.py

```diff
@@ -81,12 +81,15 @@ def deleteButtonClicked(self):
         self.librariesChanged.emit(self.libraries)
 
     def addClicked(self):
-        self.libraries.append(self.libraryNameEdit.text())
+        input = self.libraryNameEdit.text().strip()
```

Comment:
note that input is a global function in python, can you give a different name to this variable.

Reply:
Thanks for pointing that out. I have changed it. Also shouldn't we be using snake case instead of camel case for variable naming?